pkg/rtc/supervisor/participant_supervisor.go

```diff
@@ -139,55 +122,6 @@ func (p *ParticipantSupervisor) ClearPublishedTrack(trackID livekit.TrackID, pub
 	p.lock.RUnlock()
 }
 
-func (p *ParticipantSupervisor) UpdateSubscription(trackID livekit.TrackID, isSubscribe bool, sourceTrack types.MediaTrack) {
```

Comment:
Thank you @davidzhao for cleaning this up. There is a `OperationMonitorEvent` type in `interfaces.go` with events specific to subscription monitoring. Can you please delete the unused ones there also?

Reply:
got it, will do.